### PR TITLE
Set timeout as parameter

### PR DESCRIPTION
**Pull request by titilambert**
Make timeout as parameter for miio protocol
It will be useful for viomi vacuum

### CODE REVIEW

File: miio/device.py [hunk 1]
```diff
@@ -113,10 +113,13 @@ def __init__(
         start_id: int = 0,
         debug: int = 0,
         lazy_discover: bool = True,
+        timeout=5,
```
Comment (syssi) [inline miio/device.py:116]:
```suggestion
        timeout: int = 5,
```

**Review by syssi — APPROVED** [review]
Can be merged as soon the minor issue is fixed.

**Comment by titilambert** [comment]
> Can be merged as soon the minor issue is fixed.

Done ! 
Thanks !

**Comment by syssi** [comment]
Good job! Thanks! :+1:

**Pull request merged**